config/test-windows-gps.json

```diff
@@ -1,15 +1,17 @@
 {
   "version": 2,
   "robot": {
-    "drivers": ["gps", "serial.gps"],
-    "links": [["serial.gps", "gps", "raw", "raw"]],
+    "drivers": ["gps", "gps_serial"],
```

Comment:
I think we don't need this line for anything, right?

Reply:
Or rather "drivers" should be called "modules" and be of `dict` type while its keys are the names of modules instances? That is for the case when we have "links" and "modules" as top level keys in "robot".